src/Microsoft.TemplateEngine.Edge/Template/MatchKind.cs

```diff
@@ -6,6 +6,8 @@ public enum MatchKind
         Unspecified,
         Exact,
         Partial,
-        Mismatch
+        Mismatch,
+        InvalidParameterName,
```

Comment:
No, they're not worse, but pretty bad. This was made before the order of this enum mattered. I'll move InvalidParameterName above Mismatch.